infrastructure/docker/docker-compose.worker.yml

```diff
@@ -4,11 +4,19 @@ x-services-templates:
         build:
             context: services/php
             target: worker
+            cache_from:
+                - "${REGISTRY:-}/worker:cache"
         depends_on:
             - postgres
             #- rabbitmq
+        user: "${USER_ID}:${USER_ID}"
         volumes:
             - "../..:/var/www:cached"
+            - "../../.home:/home/app:cached"
+        labels:
+          - "docker-starter.worker.${PROJECT_NAME}=true"
```

Comment:
This is not needed anymore and was removed in https://github.com/jolicode/docker-starter/pull/297